app/assets/v2/images/chains/rsk.svg

```diff
@@ -1,37 +1,8 @@
-	c-0.5,0-1-0.1-1.4-0.3c-0.4-0.1-0.7-0.3-1-0.6c-0.8-0.6-1.3-1.5-1.5-2.5V90c0.2-1.4,1.1-2.6,2.4-3.1c0.4-0.2,0.9-0.3,1.4-0.3
-	c0.5,0,1,0.1,1.5,0.3c1.4,0.6,2.4,2,2.4,3.6c0,0.2,0,0.4-0.1,0.6C70.6,92.1,70.2,93.1,69.4,93.7 M70.8,124.9c0,2.2-1.7,3.9-3.9,3.9
-	c-0.7,0-1.4-0.2-2-0.6c-1.1-0.7-1.9-1.9-1.9-3.3v-0.1l0,0c0-0.4,0.1-0.8,0.2-1.2c0.5-1.3,1.6-2.3,3-2.6c0.2,0,0.4-0.1,0.7-0.1
-	c0.9,0,1.7,0.3,2.4,0.8C70.3,122.5,70.8,123.7,70.8,124.9 M64.5,116.5c-1.2,0.3-2.3,0.9-3.2,1.7l-14.8-8.6l0,0
-	c0.1-0.6,0.2-1.1,0.2-1.7c0-0.8-0.1-1.6-0.4-2.4L60.9,97c1,1,2.2,1.7,3.6,2.1L64.5,116.5L64.5,116.5z M37.9,144.9
-	c-1.9,0-3.6-1.4-3.9-3.3V141c0-1.6,1-3,2.4-3.6c1-0.4,2-0.4,3,0c1.4,0.6,2.4,2,2.4,3.6v0.6C41.4,143.5,39.8,144.9,37.9,144.9
-	 M29.3,109.6l-14.8,8.6c-0.9-0.8-2-1.4-3.2-1.7V99.1c1.4-0.4,2.6-1.1,3.6-2.1l14.6,8.4c-0.2,0.8-0.4,1.7-0.4,2.5
-	C29.1,108.5,29.2,109,29.3,109.6C29.3,109.6,29.3,109.6,29.3,109.6z M12.8,124.9c0,1.4-0.7,2.6-1.9,3.3c-0.6,0.4-1.3,0.6-2,0.6
-	c-2.2,0-3.9-1.7-3.9-3.9l0,0c0-1.2,0.6-2.3,1.5-3.1c0.7-0.5,1.5-0.8,2.4-0.8c0.2,0,0.5,0,0.7,0.1c0.6,0.1,1.2,0.4,1.7,0.8
-	c0.6,0.5,1,1.1,1.3,1.8C12.7,124.1,12.8,124.5,12.8,124.9L12.8,124.9 M4.9,90.7c0-2.2,1.7-3.9,3.9-3.9h0c0.5,0,1,0.1,1.4,0.3
-	c1.3,0.5,2.2,1.7,2.4,3.1v1.2c-0.1,1-0.7,1.9-1.5,2.5c-0.3,0.2-0.7,0.4-1,0.6c-0.4,0.2-0.9,0.3-1.4,0.3c-0.9,0-1.7-0.3-2.4-0.8
-	C5.5,93.1,5,91.9,4.9,90.7 M35.5,70.8c0.7-0.5,1.5-0.8,2.4-0.8c0.9,0,1.7,0.3,2.4,0.8c0.8,0.6,1.3,1.5,1.4,2.5v0.6
-	c0,1.2-0.6,2.3-1.5,3.1c-0.3,0.2-0.7,0.4-1,0.6c-0.9,0.4-1.9,0.4-2.8,0c-0.4-0.1-0.7-0.3-1-0.6c-1-0.7-1.5-1.9-1.5-3.1v-0.6
-	C34.2,72.3,34.7,71.4,35.5,70.8 M41.8,108c0,1.1-0.6,2.2-1.5,2.9c-0.2,0.1-0.4,0.3-0.6,0.4c-0.6,0.3-1.2,0.5-1.9,0.5
-	c-0.6,0-1.3-0.2-1.8-0.5c-0.2-0.1-0.4-0.2-0.6-0.4c-0.9-0.7-1.5-1.8-1.5-3v-0.1c0-0.3,0-0.7,0.1-1c0.2-0.8,0.7-1.5,1.3-2.1
-	c0.4-0.4,1-0.6,1.5-0.7c0.3-0.1,0.7-0.1,1-0.1c0.3,0,0.7,0,1,0.1c0.5,0.1,1,0.4,1.4,0.7c0.6,0.5,1.1,1.2,1.3,2
-	c0.1,0.4,0.1,0.7,0.1,1.1C41.7,107.8,41.8,107.9,41.8,108L41.8,108z M58.4,88.6c-0.2,0.7-0.3,1.4-0.3,2.1c0,0.7,0.1,1.5,0.3,2.2
-	l-14.7,8.5c-1-0.9-2.1-1.5-3.4-1.9V82.3c1.4-0.4,2.6-1.1,3.6-2.1L58.4,88.6z M22.7,32.1c0-12.9,11.3-22.2,15.2-25
-	c3.9,2.8,15.2,12.1,15.2,25c0,10.8-8,18.6-12.8,22.3V34c0-1.3-1.1-2.4-2.4-2.4c-1.3,0-2.4,1-2.4,2.3c0,0,0,0,0,0.1v20.4
-	C30.6,50.7,22.7,42.9,22.7,32.1 M31.8,80.2c1,1,2.2,1.7,3.6,2.1v17.2c-1.2,0.4-2.4,1-3.4,1.8l-14.8-8.5c0.4-1.4,0.4-2.8,0-4.2
-	L31.8,80.2z M17.5,126.4c0.1-0.5,0.1-1,0.1-1.5c0-0.9-0.1-1.8-0.4-2.7l14.4-8.3c1.1,1.1,2.4,1.9,3.9,2.4v16.3
-	c-1.4,0.4-2.6,1.1-3.7,2.1L17.5,126.4z M44,134.7c-1-1-2.3-1.7-3.7-2.1v-16.3c1.5-0.4,2.9-1.3,3.9-2.4l14.4,8.3
-	c-0.3,0.9-0.4,1.8-0.4,2.7c0,0.5,0,1,0.1,1.5L44,134.7z M93.6,82.9c-3.2,1.8-6.8,1.5-9.6,0.8l4.8-2.8c1.2-0.6,1.6-2.1,1-3.3
-	s-2.1-1.6-3.3-1c0,0-0.1,0.1-0.1,0.1l-4.8,2.8c0.7-3.3,2.7-6.1,5.5-7.9c4.6-2.7,10.3-1,12.9,0.1C99.6,74.4,98.3,80.2,93.6,82.9"/>
+
+<svg width="156" height="220" viewBox="0 0 156 220" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
did u mean to change this file?

Reply:
It changed on rebase I think, I didn't do it myself.

Reply:
yes the svg logo was fixed yesterday